exchanges/coinbasepro/coinbasepro.go

```diff
@@ -1748,3 +1772,21 @@ func (c *Candle) UnmarshalJSON(data []byte) error {
 	temp := [6]any{&c.Time, &c.Low, &c.High, &c.Open, &c.Close, &c.Volume}
 	return json.Unmarshal(data, &temp)
 }
+
+// UnmarshalJSON unmarshals the JSON data
+func (i *Integer) UnmarshalJSON(data []byte) error {
```

Comment:
suggestion: can be types.Number

Reply:
<img width="194" height="259" alt="image" src="https://github.com/user-attachments/assets/3b8a8641-e2dd-41ad-a7ab-a817726b3a62" />